packages/expo-av/src/AV.ts

```diff
@@ -11,9 +12,18 @@ import { Asset } from 'expo-asset';
 
 export type PlaybackSource =
   | number
-  | { uri: string; overrideFileExtensionAndroid?: string; headers?: { [fieldName: string]: string } }
+  | {
```

Comment:
Set your Prettier config to 100 cols (also set up in our .prettierrc) if it isn't already